lib/ansible/modules/network/ovs/openvswitch_bond.py

```diff
@@ -0,0 +1,361 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2019, James Denton <james.denton@outlook.com>
+# Portions copyright @ 2013 David Stygstra <david.stygstra@gmail.com>
+# Portions copyright @ 2015 VMware, Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+
+DOCUMENTATION = '''
+---
+module: openvswitch_bond
+version_added: 2.8
+author: "James Denton (@busterswt)"
+short_description: Manage Open vSwitch bonds
+requirements: [ ovs-vsctl ]
+description:
+    - Manage Open vSwitch bonds and associated options.
+options:
```

Comment:
In Ansible 2.8 we are trying to improve the module docs by including `type:` could you please add that to all the options

Reply:
Sure can, thanks for the suggestions.